libs/ktem/ktem/tags/index.py

```diff
@@ -0,0 +1,47 @@
+from ktem.index.file import FileIndex
+from ktem.index.file.base import BaseFileIndexIndexing
+
+from .pipelines import MetaIndexPipeline
+
+
+class TagIndex(FileIndex):
+    @classmethod
+    def get_admin_settings(cls):
+        from ktem.llms.manager import llms
+
+        llm_default = llms.get_default_name()
+        llm_choices = list(llms.options().keys())
+
+        settings = super().get_admin_settings()
+        settings["tags"] = {
+            "name": "Meta tags",
+            "value": "",
+            "component": "text",
+            "info": "Meta tags attached to this index (separated by comma).",
+        }
+        settings["llm"] = {
+            "name": "LLM for tagging",
+            "value": llm_default,
+            "component": "dropdown",
+            "choices": llm_choices,
+            "info": "The name of LLM model to use for tagging process.",
+        }

```

Comment:
a lot of hardcoding here, I think the settings should be initialized with some validators to fit in the llm_default and choices, and becuz we're using pydantic-settings, this is feasible